ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -880,6 +880,51 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-fips-guest-p1-f14
```

Comment:
`pN` is no longer need in test names.

Reply:
@liangxia Thanks for your remind, Could you help to review again?

Reply:
BTW, the same issue occurs multiple times.